cypress/integration/discounts/vouchers/vouchers.js

```diff
@@ -0,0 +1,101 @@
+// / <reference types="cypress"/>
+// / <reference types="../../../support"/>
+
+import faker from "faker";
+
+import { BUTTON_SELECTORS } from "../../../elements/shared/button-selectors";
+import { voucherDetailsUrl } from "../../../fixtures/urlList";
+import { createChannel } from "../../../support/api/requests/Channels";
+import * as channelsUtils from "../../../support/api/utils/channelsUtils";
+import {
+  createVoucherInChannel,
+  deleteVouchersStartsWith
+} from "../../../support/api/utils/discounts/vouchersUtils";
+import { createCheckoutWithVoucher } from "../../../support/api/utils/ordersUtils";
+import * as productsUtils from "../../../support/api/utils/products/productsUtils";
+import filterTests from "../../../support/filterTests";
+
+filterTests({ definedTags: ["all"] }, () => {
+  describe("Vouchers discounts", () => {
+    const startsWith = "CyVou-";
+    const productPrice = 100;
+    const shippingPrice = 100;
+
+    let defaultChannel;
+    let createdChannel;
+    let shippingMethod;
+    let variants;
+    let product;
+    let address;
+
+    before(() => {
+      cy.clearSessionData().loginUserViaRequest();
+      channelsUtils.deleteChannelsStartsWith(startsWith);
+      deleteVouchersStartsWith(startsWith);
+      const name = `${startsWith}${faker.datatype.number()}`;
+      productsUtils
+        .createProductWithShipping({ name, productPrice, shippingPrice })
+        .then(
+          ({
+            variantsList: variantsResp,
+            defaultChannel: channel,
+            shippingMethod: shippingMethodResp,
+            address: addressResp,
+            product: productResp
+          }) => {
+            variants = variantsResp;
+            defaultChannel = channel;
+            shippingMethod = shippingMethodResp;
+            address = addressResp;
+            product = productResp;
+            createChannel({ name });
+          }
+        )
+        .then(channel => {
+          createdChannel = channel;
+        });
+    });
+
+    it("should delete voucher", () => {
```

Comment:
Also I wonder why we have double tests for delete voucher. 
We have this test here and   https://github.com/saleor/saleor-dashboard/pull/1817/files 
 -> cypress/integration/discounts/vouchers/updateVouchers.js

Reply:
@karola312 I did not mention about  cypress/integration/discounts/vouchers/vouchers.js 
I mention about `updateVoucher.js`. There is a difference between `voucher.js` vs `updateVoucher.js`

Reply:
![image](https://user-images.githubusercontent.com/29308910/152303207-1eeca3e9-07d5-4a03-a840-58408a709b9b.png)

Reply:
So if it is about updateVoucher from next PR, I will take care of it in next PR